.ci/install_virtiofsd.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+
+main() {
+	local buildscript="${katacontainers_repo_dir}/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh"
+
+	# Just in case the kata-containers repo is not cloned yet.
+	clone_katacontainers_repo
+
+	pushd $katacontainers_repo_dir
+	sudo -E PATH=$PATH bash ${buildscript} --build=virtiofsd
+	sudo tar xvJpf build/kata-static-virtiofsd.tar.xz -C /
```

Comment:
Nit: I know `tar` can handle this, but for consistency with `-C` I'd consider adding the dash:

```suggestion
	sudo tar -xvJpf build/kata-static-virtiofsd.tar.xz -C /
```

Reply:
Updated here, and also in the `install_cloud_hypervisor.sh` script.